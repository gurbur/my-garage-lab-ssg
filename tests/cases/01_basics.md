# 가장 큰 제목 (H1)

This is first paragraph. 마크다운의 문단은 한 개 이상의 연속된 텍스트 줄로 구성됩니다.

이렇게 빈 줄을 사이에 두면, 새로운 문단이 시작됩니다.

## 중간 제목 (H2)

*이텔릭체*와 **볼드체**, 그리고 ***이텔릭과 볼드를 동시에*** 사용할 수 있습니다. `인라인 코드`도 처리해야 합니다.

---

### 작은 제목 (H3)

수평선 위아래로 다른 내용이 올 수 있습니다.
