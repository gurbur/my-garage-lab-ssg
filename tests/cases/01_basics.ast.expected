-> DOCUMENT
	-> HEADING1
		-> TEXT | data1: "가장 큰 제목 (H1)"
	-> PARAGRAPH
		-> TEXT | data1: "This is first paragraph. 마크다운의 문단은 한 개 이상의 연속된 텍스트 줄로 구성됩니다."
	-> PARAGRAPH
		-> TEXT | data1: "이렇게 빈 줄을 사이에 두면, 새로운 문단이 시작됩니다."
	-> HEADING2
		-> TEXT | data1: "중간 제목 (H2)"
	-> PARAGRAPH
		-> ITALIC | data1: "이텔릭체"
		-> TEXT | data1: "와 "
		-> BOLD | data1: "볼드체"
		-> TEXT | data1: ", 그리고 "
		-> ITALIC_AND_BOLD | data1: "이텔릭과 볼드를 동시에"
		-> TEXT | data1: " 사용할 수 있습니다. "
		-> CODE | data1: "인라인 코드"
		-> TEXT | data1: "도 처리해야 합니다."
	-> LINE
	-> HEADING3
		-> TEXT | data1: "작은 제목 (H3)"
	-> PARAGRAPH
		-> TEXT | data1: "수평선 위아래로 다른 내용이 올 수 있습니다. "
